recipes/eulexistdb/meta.yaml

```diff
@@ -1,17 +1,12 @@
-{% set name = "eulexistdb" %}
 {% set version = "0.21.1" %}
-{% set file_ext = "tar.gz" %}
```

Comment:
@marcelotrevisani I'm working on conda-skeleton to remove this excess of jinja variables. It is useless and silly. (Only the `version` is useful and sometimes the `name` if people are copying and pasting recipes to create a new one.)

Reply:
Thanks for let me know! :)